source/dashboards/dashboards.markdown

```
@@ -27,6 +31,7 @@ Screenshot of the Dashboard list.
 
 Home Assistant ships with some dashboards out of the box:
 
+- [Areas dashboard (experimental)](#area-dashboard-experimental)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)